### DISCUSSION TITLE
Is generatePublisherEvidence still a thing for .NET 5+?

### DISCUSSION THREAD

**Discussion by jwfx** [dotnet/runtime#64703] [Q&A] [answered]
For our .NET Framework 4.8 application we had to disable OCSP checks via `generatePublisherEvidence` because the checking was taking so long that Windows was killing the service during startup.
Similar behavior is described here: https://github.com/dotnet/docs/issues/7351

Since it is very hard to find actual information about this topic, here a few questions:

- Is that OCSP checking still done with .NET 5+ applications?
- Is putting `<generatePublisherEvidence enabled="false"/>` into `app.config` still the way to go to disable this behavior?

**Comment by jkotas** [comment, marked as answer]
Publisher evidence is part of the Code Access Security feature in .NET Framework. Code Access Security feature was completely removed in .NET Core/5+. 

> Is that OCSP checking still done with .NET 5+ applications?

No.

> Is putting <generatePublisherEvidence enabled="false"/> into app.config still the way to go to disable this behavior?

This setting has no effect for .NET Core/5+ applications.
